keyboards/retro_75/readme.md

```diff
@@ -0,0 +1,15 @@
+# Retro75
+
+![Retro75](https://i.imgur.com/spS7PHo.jpg)
+
+A 75% ARM STM32F072 keyboard, with WS2812 LEDs.
+
+Keyboard Maintainer: [zvecr](https://github.com/zvecr)  
```

Comment:
```suggestion
* Keyboard Maintainer: [zvecr](https://github.com/zvecr)
```

Reply:
Shoot, forgot about that in the template cleanup.